about.html

```diff
@@ -12,53 +12,76 @@
 </head>
 
 <body>
+  <header>
```

Comment:
The image is above the nav items, if you change the image to be inside the nav it could fix this issue.